Makefile

```diff
@@ -14,6 +14,8 @@ build:
 	export LD_LIBRARY_PATH="${LD_LIBRARY_PATH2}"; mvn -B package
 	cp -r ${CAFFE_ON_SPARK}/caffe-public/python/caffe ${CAFFE_ON_SPARK}/caffe-grid/src/main/python/
 	cd ${CAFFE_ON_SPARK}/caffe-grid/src/main/python/; zip -r caffeonsparkpythonapi  *; mv caffeonsparkpythonapi.zip ${CAFFE_ON_SPARK}/caffe-grid/target/;cd ${CAFFE_ON_SPARK}
+	cp ${CAFFE_ON_SPARK}/caffe-grid/src/test/python/PythonTest.sh ${CAFFE_ON_SPARK}/data/
+	cd data; ./PythonTest.sh; cd ..	
```

Comment:
Why are we moving files? We should use the solver and network files in test/resources folder, not the data folder.

Also, where is PythonTest.sh file? You may have missed it.